### PR TITLE
fix: avatar design updates

### PR DESCRIPTION
**Pull request by droshev**
The Avatar got few design updates:
- border color theming parameter
- badge color and background theming parameters

Before:
Screen Shot 2020-08-17 at 6 14 00 PM
Screen Shot 2020-08-17 at 6 13 42 PM
Screen Shot 2020-08-19 at 11 18 37 PM

After:
Screen Shot 2020-08-17 at 6 14 08 PM
Screen Shot 2020-08-17 at 6 13 48 PM
Screen Shot 2020-08-19 at 11 13 36 PM

### CODE REVIEW

File: src/avatar.scss [hunk 1]
```diff
@@ -49,6 +48,7 @@ $block: #{$fd-namespace}-avatar;
       background-size: cover;
       background-position: 50%;
       border-radius: var(--sapElement_BorderCornerRadius);
+      box-sizing: border-box;
```
Comment (InnaAtanasova) [inline src/avatar.scss:51]: Why did you add this? It's coming with the reset function.
Screen Shot 2020-08-19 at 10 34 09 AM

File: src/avatar.scss [hunk 2]
```diff
@@ -122,14 +122,16 @@ $block: #{$fd-namespace}-avatar;
   &__zoom-icon {
     @include fd-flex-center();
 
-    background-color: var(--sapLinkColor);
-    color: var(--sapContent_ContrastIconColor);
+    background-color: var(--sapButton_Emphasized_Background);
+    box-sizing: border-box;
```
Comment (InnaAtanasova) [inline src/avatar.scss:126]: same here

**Comment by droshev** [comment]
> LGTM, I noticed that avatar in shellbar got different border
> image
> But I guess it's not in scope of this PR

that is a valid comment - I addressed that too

File: src/avatar.scss (hunk 2 above)
Comment (droshev) [reply src/avatar.scss]: thanks! I removed it

File: src/avatar.scss (hunk 1 above)
Comment (droshev) [reply src/avatar.scss]: thanks! I removed it
